openapi.yaml

```diff
@@ -301,61 +273,79 @@ paths:
     get:
       operationId: getRasSearchRuns
       summary: Get Runs from Query
+      description: |
+        API endpoint to query the Result Archive Store (RAS) for a list of runs based on the given search criteria.
+
+        If no search criteria is provided then the first page of resutls will be returned. 
+
+        If no timeframe is specified in the query then the results will update as new tests are run which may result in duplicate and/or missing test runs.
```

Comment:
Wondering if we should also add that query parameters can be repeated ?
eg: runid=xxx&runid=yyy ?

Can they be repeated ? Which ones ? Perhaps we should mention it on all those which can be repeated ?

Reply:
Only sort is repeatable at the moment. Will be worth while to raise more issues to explore the possibility of partial querying and to expand the sorting capabillties.

I have updated the document to reflect the current sort functionality.